dace/libraries/blas/environments/intel_mkl.py

```diff
@@ -7,12 +8,11 @@ class IntelMKL:
     cmake_minimum_version = None
     cmake_packages = ["BLAS"]
     cmake_variables = {"BLA_VENDOR": "Intel10_64lp"}
-    cmake_includes = []  # For some reason, FindBLAS does not find includes
+    cmake_includes = [os.path.join(os.environ['MKLROOT'], 'include')]
```

Comment:
I would add another function called `_find_mkl_include()`, which returns `[path.join(MKLROOT, include)]` if MKLROOT is in `os.environ`, or `[]` otherwise.